guide/guide-random.rst

```diff
@@ -58,11 +74,43 @@ full-rank for the given dimension.
 
 For example,
 
-.. ipython::
+.. doctest:: [random]
+  :skipif: True
+
+   >>> rand_dm(5, density=0.5)
+   Quantum object: dims = [[5], [5]], shape = (5, 5), type = oper, isherm = True
```

Comment:
In the case of testing a random generation, you could also try to set a random seed (in a hidden block) and then test it.